airbyte-local.sh

```diff
@@ -348,6 +348,16 @@ function parseStreamPrefix() {
             # Remove first and last elements
             src_docker_image_parts=("${src_docker_image_parts[@]:1}")
             src_docker_image_parts=("${src_docker_image_parts[@]::${#src_docker_image_parts[@]}-1}");
+            if [[ -z "$connection_name" ]]; then
```

Comment:
why are we inferring instead of requiring `connection_name` to be provided?

Reply:
Why not? We would just tell them to pass the feed name anyway. It's hard to justify all the options required by our feeds and the CLI. This is a major pain point when asking customers to run connectors.